Resources/Locale/en-US/_Impstation/popup/buckettrap.ftl

```diff
@@ -0,0 +1 @@
+bucket-propped = You carefully prop the bucket atop of the door.
```

Comment:
should be "atop the door" or "on top of the door", and needs a trailing newline at the end of the file